uw-frame-components/js/frame-config.js

```diff
@@ -1069,5 +1070,11 @@ define(['angular'], function(angular) {
             'description': 'This flag disables announcement group filtering ' +
               'if you have it enabled (page refresh required)',
           },
+          {
+            'id': 'optInAvatar',
+            'title': 'Allow Use Of My Photo',
+            'description': 'This flag allows use of your photograph ' +
+              'if you have it enabled (page refresh required)',
+          },
```

Comment:
This method of opting in would require the user to know about and navigate to our nuts and bolts settings page. We should keep this feature flag, but maybe add a toggle in the user settings area?